.github/workflows/release.yml

```diff
@@ -0,0 +1,59 @@
+# Copyright 2024 JanusGraph Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: janusgraph-python-deployment
+
+on: workflow_dispatch
```

Comment:
I'd suggest that we change this to:
```yml
release:
    types:
      - published
```

That way we can simply create a new release in the GitHub UI and it will then trigger this workflow to actually deploy the package.

Reply:
Thank you for the great suggestion. I have fixed this and squashed the commits.

Reply:
Sorry, I should have been more clear with my suggestion, because I already omitted the `on:`, but I think that it's still needed. So, this should really look like:

```yaml
on:
  release:
    types:
      - published
```

For reference, here's the part of the GitHub docs on this: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release
Only difference there is that they put `published` in a list like this: `[published]` which is the same as putting it in a new line with `-` in front.

Reply:
Thanks Florian for the clarification. I have fixed it.